### DISCUSSION TITLE
Is there a reentrant Bitwuzla version/compilation?

### DISCUSSION THREAD

**Discussion by baierd** [bitwuzla/bitwuzla#15] [Q&A] [answered]
Hello,
i was wondering if it is possible to compile Bitwuzla such that it is useable reentrantly?

Thank you for your work and help!

**Comment by mpreiner** [comment, marked as answer]
Hi Daniel,

Bitwuzla should be reentrant already. Did you run into any issues?

**Comment by baierd** [comment]
No i just didn't see any description stating if it is or not. Maybe i missed it?
Anyway, thats perfect. Thank you!
